src/Image.php

```diff
@@ -132,6 +132,66 @@ public function deleteSourceImage($hash, $organization = '')
         return '204' == $response->getStatusCode();
     }
 
+    /**
+     * Restore a source image.
+     *
+     * @param string $hash         Hash of the image
+     * @param string $organization Optional organization name
+     *
+     * @throws GuzzleException If the request fails for a different reason than image not found
+     *
+     * @return bool True if successful, false if image not found
+     */
+    public function restoreSourceImage($hash, $organization = '')
+    {
+        try {
+            $response = $this->call('POST', implode('/', [self::SOURCEIMAGE_RESOURCE, $this->getOrganization($organization), $hash, 'restore']));
+        } catch (GuzzleException $e) {
+            if (404 == $e->getCode()) {
+                return false;
+            }
+
+            throw $e;
+        }
+
+        return '200' == $response->getStatusCode();
+    }
+
+    /**
+     * Copy a source image to another org.
+     *
```

Comment:
should this explain the permission requirements?

Reply:
added